src/com/trollworks/gcs/spell/Spell.java

```diff
@@ -728,7 +729,12 @@ public SkillAttribute getAttribute() {
 
     /** @return Whether this is a "Very Hard" spell or not. */
     public boolean isVeryHard() {
-        return mIsVeryHard;
+        return mDifficulty == SkillDifficulty.VH;
+    }
```

Comment:
`isVeryHard()` and all of its uses needs to be removed and replaced with checking the new difficulty value instead